aepsych/strategy.py

```diff
@@ -174,34 +174,41 @@ def normalize_inputs(self, x, y):
         """converts inputs into normalized format for this strategy
 
         Args:
-            x (np.ndarray): training inputs
-            y (np.ndarray): training outputs
+            x (torch.Tensor): training inputs
+            y (torch.Tensor): training outputs
 
         Returns:
-            x (np.ndarray): training inputs, normalized
-            y (np.ndarray): training outputs, normalized
+            x (torch.Tensor): training inputs, normalized
+            y (torch.Tensor): training outputs, normalized
             n (int): number of observations
         """
         assert (
             x.shape == self.event_shape or x.shape[1:] == self.event_shape
         ), f"x shape should be {self.event_shape} or batch x {self.event_shape}, instead got {x.shape}"
-
+        
+        # Handle scalar y values
+        if y.ndim == 0:
+            y = y.unsqueeze(0)
+        
         if x.shape == self.event_shape:
             x = x[None, :]
 
         if self.x is None:
-            x = np.r_[x]
+            x = x
```

Comment:
Rewrite this please to just be `if self.x is not None: ... `, same with the equivalent below, it shouldn't just be assigning to itself.